### PR TITLE
Ignore NoVerbSelected commandline parsing error

### LINKED ISSUE #136: Commandline parsing with no verb shows "no verb selected" error
When running the tool with just `wingetcreate`, a "no verb selected" error is displayed to the user. 

A user should not be shown a warning for running the tool without any verbs as they could be running it to learn more about the tool's commands/functionality.

Expected behavior:
No warning message shown for just running `wingetcreate`

### PR DESCRIPTION
**Pull request by ryfu-msft**
Fixes #136 

When running `wingetcreate` with no verb, the tool will display a warning stating a NoVerbSelected error. 

This PR adds changes to ignore that specific parsing error as a user should not be shown a warning for not selecting a verb and should instead be shown the help screen.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-create/pull/135)

### CODE REVIEW

File: src/WingetCreateCLI/Program.cs [hunk 1]
```diff
@@ -110,12 +110,16 @@ private static void DisplayHelp(NotParsed<object> result)
 
         private static void DisplayParsingErrors<T>(ParserResult<T> result)
         {
-            var builder = SentenceBuilder.Create();
-            var errorMessages = HelpText.RenderParsingErrorsTextAsLines(result, builder.FormatError, builder.FormatMutuallyExclusiveSetErrors, 1);
-
-            foreach (var error in errorMessages)
+            NotParsed<object> notParsed = result as NotParsed<object>;
```
Comment (palenshus) [inline src/WingetCreateCLI/Program.cs:113]: Why don't you just change the function parameter to be NotParsed<object>

**Review by palenshus — CHANGES_REQUESTED** [review]
:clock1:

File: src/WingetCreateCLI/Program.cs (hunk 1 above)
Comment (ryfu-msft) [reply src/WingetCreateCLI/Program.cs]: Changed function parameter to NotParsed

Comment (ryfu-msft) [reply src/WingetCreateCLI/Program.cs]: Changed function parameter to NotParsed

**Review by palenshus — APPROVED** [review]

**Pull request merged**